bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/extension/Extension.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.extension;
+
+/**
+ * This class defines an extension.
+ *
+ * @author Kai Kreuzer - Initial contribution and API
+ *
+ */
+public class Extension {
```

Comment:
If not should we add getters and setters? It´s not that I strictly want to have it, but all the other core domain classes have it (Thing, Channel, etc.)